README.rst

```diff
@@ -50,55 +50,89 @@ Currently available datasets
 Future datasets
 ~~~~~~~~~~~~~~~
 
-* Raw FERC Form 1 DB (SQL) – `browse DB online <https://data.catalyst.coop/ferc1>`__
-* PUDL DB (SQL) – `browse DB online <https://data.catalyst.coop/pudl>`__
-* Census Demographic Profile 1 (SQL)
+* Raw FERC Form 1 DB (SQLite) -- `browse DB online <https://data.catalyst.coop/ferc1>`__
+* PUDL DB (SQLite) -- `browse DB online <https://data.catalyst.coop/pudl>`__
+* Census Demographic Profile 1 (SQLite)
 
 Ongoing Development
 -------------------
 
 Development is currently being organized under these epics in the main
 PUDL repo:
 
+* `Intake SQLite Driver <https://github.com/catalyst-cooperative/pudl/issues/1156>`__
 * `EPA CEMS Intake Catalog <https://github.com/catalyst-cooperative/pudl/issues/1564>`__
-* `Prototype SQLite Intake Catalog <https://github.com/catalyst-cooperative/pudl/issues/1156>`__
+* `PUDL Intake Catalog <https://github.com/catalyst-cooperative/pudl/issues/1179>`__
 
-See the `issues in this repository
+See the `issues in the pudl-catalog repository
 <https://github.com/catalyst-cooperative/pudl-catalog/issues>`__ for more
 detailed tasks.
 
-Planned data distribution system
+Usage
+-----
+
+Public data and "requester pays"
```

Comment:
can we add this stuff into the data access page instead of the readme??

Reply:
I'll break it out into a separate page and link to it. I agree this is too much detail for the README.